dialog_text.go

```diff
@@ -26,12 +26,13 @@ type TextInputElement struct {
 }
 
 // NewTextInput constructor for a `text` input
-func NewTextInput(name, label, text string) *TextInputElement {
+func NewTextInput(name, label, text string, optional bool) *TextInputElement {
```

Comment:
prefer `options ...TextInputOption` instead of just concating fields to the function signature.

```
type TextInputOption func(*TextInputOption)
var options = []TextInputOption{}
t := &TextInputElement{}
for _, opt := range options {
  opt(&t)
}
```

Reply:
Thank you for your comment! Can you say me if i understood your idea with my last commit? (i'm new in language).